packages/apputils/src/instancetracker.ts

```diff
@@ -320,7 +326,7 @@ export class InstanceTracker<T extends Widget>
    */
   inject(widget: T): void {
     Private.injectedProperty.set(widget, true);
-    this.add(widget);
+    void this.add(widget);
```

Comment:
I initially made this async, but the note indicates that injected widgets are not saved, so we don't need to wait on that.